lib/Doctrine/Common/Cache/ClearableCache.php

```diff
@@ -29,7 +32,7 @@
 interface ClearableCache
 {
     /**
-     * Deletes all cache entries.
+     * Deletes all cache entries in the current cache namespace.
```

Comment:
Note that namespaces are a specific feature of some cache types, not necessarily of a `ClearableCache`